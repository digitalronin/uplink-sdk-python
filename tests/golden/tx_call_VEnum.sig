r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 49701065736551789958327503597141348774163841267056295629727687132074932788877
