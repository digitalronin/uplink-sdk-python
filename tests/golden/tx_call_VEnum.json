{
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "header": {
        "tag": "TxContract", 
        "contents": {
            "tag": "Call", 
            "contents": {
                "args": [
                    {
                        "tag": "VEnum", 
                        "contents": "Foo"
                    }
                ], 
                "method": "get", 
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE4xMDMxMDQzMTE2Nzg0MTc2ODYxMDk0MTY5MjQ1Mzg5Njk0MjMyNDAzNTc2Mzc0OTMzNDI3MDA3NzUxNjc1MTEzNTk2NzkxOTUwNjIyODI="
}